kirin/piv/model_maker.py

```diff
@@ -255,6 +255,9 @@ def build_trip_updates(self, rt_update):
 
         dict_objects = get_value(json, "objects")
         json_train = get_value(dict_objects[0], "object")  # TODO: can we get more than 1 relevant in objects[]?
+        brand_code = json_train.get("marque", {}).get("code", "")
```

Comment:
To be uniform with the tests below. see L293
```suggestion
        brand_code = get_value(get_value(json_train, "marque"), "code", nullable=True)
```